airflow/contrib/operators/druid_operator.py

```diff
@@ -34,10 +34,11 @@ def __init__(
         self,
         json_index_file,
         druid_ingest_conn_id='druid_ingest_default',
+        max_ingestion_time=18000,
```

Comment:
Why the default?

Reply:
This was already the default in the hook. Five hours is a reasonable time, assuming you do daily delta's.

Reply:
Maybe it is better to have no max if you don't explicitly supply it

Reply:
I think that would be indeed the best